index.html

```diff
@@ -1,354 +1,369 @@
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="./docs/index.html" target="_blank"><p style="font-size:2vw;">Docs</p></a>
-          </li>
-          <!-- <li class="nav-item">
-            <a class="nav-link" href="https://data.posydon.org" target="_blank"><p style="font-size:2vw;">Database</p></a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="https://posydon.odahub.io" target="_blank"><p style="font-size:2vw;">Webapp</p></a>
-          </li> -->
-        </ul>
-      </div>
-    </nav>
-    
-    <div id="demo" class="carousel slide" data-ride="carousel">
-      <ul class="carousel-indicators">
-        <li data-target="#demo" data-slide-to="0" class="active"></li>
-        <li data-target="#demo" data-slide-to="1"></li>
-      </ul>
-      <div class="carousel-inner">
-        <div class="carousel-item active">
-          <img class="d-block mx-auto" src="images/carousel/webpage_photos.006.jpeg" alt="" style="width:100%; height: auto">
-          <div class="carousel-caption">
-            <p style="font-size:1.2vw;"><a href="https://ciera.northwestern.edu/gallery/binary-system-mass-transfer/" target="_blank" style="color: rgb(255,255,255)">Credit: NU Viz and CIERA - Matthew McCrory, Francesca Valsecchi, and Vicky Kalogera</a></p>
-          </div>
-        </div>
-        <div class="carousel-item">
-          <img class="d-block mx-auto" src="images/carousel/webpage_photos.005.jpeg" alt="" style="width:100%; height: auto">
-          <div class="carousel-caption">
-            <p style="font-size:1.2vw;"><a href="https://ciera.northwestern.edu/gallery/black-hole-accretion-disk/" target="_blank" style="color: rgb(255,255,255)">Credit: NU Viz and CIERA - Matthew McCrory, Francesca Valsecchi, and Vicky Kalogera</a></p>
-          </div>
-        </div>
-      </div> 
-      <!-- Left and right controls -->
-      <a class="carousel-control-prev" href="#demo" data-slide="prev">
-        <span class="carousel-control-prev-icon"></span>
-      </a>
-      <a class="carousel-control-next" href="#demo" data-slide="next">
-        <span class="carousel-control-next-icon"></span>
-      </a>
-    </div>
-
-    <div class="container p-4">
-      <div class="row">
-        <div class="col-md-7 bg-light text-dark">
-          <p style="position:relative;">
-            <a id="About" style="position:absolute; top:-180px;"></a>
-          </p>
-          <h1 style="text-align:center">ABOUT</h1> 
-          <hr/>
-          <p><b>POSYDON</b> : <b>PO</b>pulation <b>SY</b>nthesis with <b>D</b>etailed binary-evolution simulati<b>ON</b>s</p>
-          <p><i>A next-generation single and binary-star population synthesis code incorporating full stellar structure and evolution modeling with the use of <a href="http://mesa.sourceforge.net/" target="_blank">MESA</a></i>.</p>
```

Comment:
Just a suggestion, it might be nice to repeat the same intro that's in the v2 docs at the top of the About section :

"POSYDON is a next-generation single and binary-star population synthesis tool. Our vision is to provide researchers and enthusiasts with a state-of-the-art platform to delve into the intricacies of stellar structures and binary evolutions using MESA. With full stellar structure modeling, advanced machine learning techniques, and a modular architecture, POSYDON stands at the forefront of astrophysical simulations."

Reply:
General rewriting suggestion (after the potential intro above):

"POSYDON is being developed by a _collaborative team [link to team page?]_ of astrophysicists and computer scientists, led by the Principal Investigators Tassos Fragos (Universit&#233; de Gen&#232;ve) and Vicky Kalogera (Northwestern University)."

The POSYDON code is modular in many aspects. For general population synthesis, users can specify initial population properties and adopt choices that determine how stellar evolution proceeds. Populations are simulated using POSYDON's library of pre-computed grids, which contain MESA evolutionary tracks for single, non-interacting, and interacting binaries. Users are also welcome to build their own library of MESA grids for use with POSYDON. Machine-learning methods are incorporated and applied on the pre-computed grids for classification and various interpolation calculations. There is also architecture available for the development of irregular grids guided by active learning for computational efficiency.

POSYDON is a a publicly available tool, and we invite its use by everyone in the astronomy community. Please use the tabs above to: access the _source code [link to code page?]_; find extensive _documentation [link to docs page?]_ on installing and using POSYDON; download the _grids [link to zenodo home page?]_ of single- and binary-star evolutionary models; and learn how to perform small-scale _population synthesis simulations [link to relevant tutorials page?]_ with POSYDON."

Reply:
Updated with suggested re-write.